proto/ibc/connection/query.proto

```diff
@@ -4,19 +4,26 @@ package ibc.connection;
 import "gogoproto/gogo.proto";
 import "cosmos/base/query/v1beta1/pagination.proto";
 import "ibc/connection/connection.proto";
+import "google/api/annotations.proto";
 
 option go_package = "github.com/cosmos/cosmos-sdk/x/ibc/03-connection/types";
 
 // Query provides defines the gRPC querier service
 service Query {
   // Connection queries an IBC connection end.
-  rpc Connection(QueryConnectionRequest) returns (QueryConnectionResponse) {}
+  rpc Connection(QueryConnectionRequest) returns (QueryConnectionResponse) {
+		option (google.api.http).get = "/cosmos/ibc/connection/v1beta1/connection/{connection_id}";
```

Comment:
yes, it would be nice

Reply:
Cool, then we don't need to change these annotations here

Reply:
> Cool, then we don't need to change these annotations here

I would suggest just removing the `/cosmos` prefix, so that path and proto package name are consistent.